source/support/data-protection/index.html.md.erb

```diff
@@ -0,0 +1,162 @@
+---
+title: Data Protection for GOV.UK Account users
+weight: 32
+last_reviewed_on: 2021-02-02
+review_in: 6 months
+---
+
+# Respond to data protection requests
+
+When we use and hold user data, users gain rights under the UK General Data Protection Regulation (UK GDPR).
+Users make requests to exercise those rights and we have responsibilities to respond.
+
+Below are some of the scenarios we will have to to understand and respond to along with information on who
+has responsibilities.
+
+Responding to requests may involve the GOV.UK Accounts team, GOV.UK User support and the GDS Privacy office.
+
+This guide is not an complete list of everything we must do under the UK GDPR. Instead it focuses on user rights
+requests and how to respond.
+
+# Who handles requests?
```

Comment:
Each page of content must only have one H1.

Reply:
Aktch, that was stupid of me. Great thanks have moved all the headings down a level.
The title becomes a H1 right?

Reply:
Correct - the title "Respond to data protection requests" becomes the sole H1